.github/workflows/build-test.yml

```diff
@@ -353,3 +354,11 @@ jobs:
       - name: Build and test
         run: |
           ctest -j 2 -VV -S dashboard.cmake
+
+      - name: Publish Documentation html as GitHub Artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: Documentation
+          path: |
+            /home/runner/work/ITKExamples/build/ITKExamples-5.2.0-html.tar.gz
+            /home/runner/work/ITKExamples/build/ITKExamples-5.2.0-html.zip
```

Comment:
Can we use [`GITHUB_WORKSPACE`](https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables) here? It is [referenced with `${{ github.workspace }}`](https://github.com/actions/upload-artifact#environment-variables-and-tilde-expansion)